File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/create_processing_job_packages_request_item.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class CreateProcessingJobPackagesRequestItem(TypedDict, total=False):
    ExpirationDate: str
    FinishDate: str
    FinishNote: str
    FinishProcessingJob: bool
    IsFinishedGood: bool
    Item: str
    JobName: str
    Location: str
    Note: str
    PackageDate: str
    PatientLicenseNumber: str
    ProductionBatchNumber: str
    Quantity: float
    SellByDate: str
    Sublocation: str
    Tag: str
    UnitOfMeasure: str
    UseByDate: str
    WasteCountQuantity: float
    WasteCountUnitOfMeasureName: str
    WasteVolumeQuantity: float
    WasteVolumeUnitOfMeasureName: str
    WasteWeightQuantity: float
    WasteWeightUnitOfMeasureName: str

File: sdks/thunkmetrc-ts/wrapper/src/models/CreateProcessingJobPackagesRequestItem.ts

export interface CreateProcessingJobPackagesRequestItem {
    ExpirationDate?: string;
    FinishDate?: string;
    FinishNote?: string;
    FinishProcessingJob?: boolean;
    IsFinishedGood?: boolean;
    Item?: string;
    JobName?: string;
    Location?: string;
    Note?: string;
    PackageDate?: string;
    PatientLicenseNumber?: string;
    ProductionBatchNumber?: string;
    Quantity?: number;
    SellByDate?: string;
    Sublocation?: string;
    Tag?: string;
    UnitOfMeasure?: string;
    UseByDate?: string;
    WasteCountQuantity?: number;
    WasteCountUnitOfMeasureName?: string;
    WasteVolumeQuantity?: number;
    WasteVolumeUnitOfMeasureName?: string;
    WasteWeightQuantity?: number;
    WasteWeightUnitOfMeasureName?: string;
}
